src/supporting-data/citations/components/__tests__/LiteratureCitation.spec.tsx

```diff
@@ -30,18 +30,18 @@ describe('Publication component', () => {
     const mockData: CitationsAPIModel = {
       citation: {
         id: '14702039',
-        authors: ['Ota T.'],
+        authors: ['Smith X.'],
       },
     };
     rendered = customRender(<LiteratureCitation data={mockData} />);
-    const author = await screen.findByText('Ota T.');
-    expect(author).toBeTruthy();
+    const author = await screen.findByText('Smith X.');
```

Comment:
Is there a specific need for these `await ... findBy` instead of a plain `getBy`? When reading the test it feels like there is some asynchronous rendering happening, which is not the case, is it?
See https://testing-library.com/docs/queries/about/#types-of-queries for reference

Reply:
No need, I think I had a `fireEvent` in there before and forgot to update it.

Reply:
I went ahead and modified it in the others places where this was used too, and modified a bit more the rest of the test file